packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -794,7 +793,11 @@ class ComboBoxInternal extends React.Component<IComboBoxInternalProps, IComboBox
       const items = currentOptions
         .map((item, index) => ({ ...item, index }))
         .filter(
-          option => isNormalOption(option) && getPreviewText(option).toLocaleLowerCase().indexOf(updatedValue) === 0,
+          option =>
```

Comment:
This change and the one later in the file look like they're from prettier being out of date. Try running `yarn`, reload VS Code, and save the file again (after making some minor change like adding a space).